account_asset_management/models/account_asset_profile.py

```diff
@@ -207,11 +204,12 @@ def _compute_prorrata(self):
             if profile.method_time != "year":
                 profile.prorata = True
 
-    @api.model
-    def create(self, vals):
-        if vals.get("method_time") != "year" and not vals.get("prorata"):
-            vals["prorata"] = True
-        profile = super().create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get("method_time") != "year" and not vals.get("prorata"):
+                vals["prorata"] = True
+        profile = super().create(vals_list)
```

Comment:
This returns several records. Take it into account.

Reply:
i think its ok now